src/app/components/community/link-game-modal/link-game-modal.ts

```diff
@@ -0,0 +1,69 @@
+import { Component, Prop } from 'vue-property-decorator';
+import { propRequired } from '../../../../utils/vue';
+import { Api } from '../../../../_common/api/api.service';
+import { Community } from '../../../../_common/community/community.model';
+import { Game } from '../../../../_common/game/game.model';
+import AppGameThumbnailImg from '../../../../_common/game/thumbnail-img/thumbnail-img.vue';
+import { Growls } from '../../../../_common/growls/growls.service';
+import AppLoading from '../../../../_common/loading/loading.vue';
+import { BaseModal } from '../../../../_common/modal/base';
+import { AppTooltip } from '../../../../_common/tooltip/tooltip';
+
+@Component({
+	components: {
+		AppLoading,
+		AppGameThumbnailImg,
+	},
+	directives: {
+		AppTooltip,
+	},
+})
+export default class AppCommunityLinkGameModal extends BaseModal {
+	@Prop(propRequired(Community)) community!: Community;
+
+	page = 0;
+	isLoading = true;
+	lastPage = false;
+	games: Game[] = [];
+
+	mounted() {
+		this.loadPage();
+	}
+
+	async loadPage() {
+		this.isLoading = true;
+
+		try {
+			const payload = await Api.sendRequest(
+				'/web/dash/communities/games/list/' + this.community.id + '/' + this.page,
```

Comment:
super minor, but we rarely use pages as path parts, and our pages usually are 1-based.